src/utils/utils.go

```diff
@@ -78,18 +78,18 @@ func SetFileInIgnition(ignitionData []byte, filePath, fileContents string, mode
 		return nil, err
 	}
 
+	rootUser := "root"
 	file := types.File{
 		Node: types.Node{
-			Filesystem: "root",
-			Path:       filePath,
-			Overwrite:  nil,
-			Group:      nil,
-			User:       nil,
+			Path:      filePath,
+			Overwrite: nil,
```

Comment:
@tsorya , @yevgeny-shnaidman what do you think about moving this to the ignition generator?
We do similar things there

Reply:
good idea, we can do it once @mhrivnak code is delivered, and we can gather all related ignition code at one place

Reply:
agree